stories/carousel/carousel.stories.js

```diff
@@ -477,3 +476,133 @@ carouselNoNavigation.parameters = {
 `
     }
 };
+
+export const horizontalCarousel = () => `<div style="display: flex; flex-direction: column; align-items: center; background: #CCD1D1;">
+    <h4>Items in horizontal direction</h4>
+    <div
+        class="fd-carousel"
+        data-ride="carousel"
+        style="margin-bottom: 3rem; max-width: 60rem;">
+        <div class="fd-carousel__content fd-carousel__content--horizontal" style="text-align: center; padding: 1rem; min-height: 15.5rem;">
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <div
+                    style="
+                        width:100%;
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
+                        height: 15.5rem;
+                        background-size:cover;"
+                    role="img"
+                    aria-label="Carousel Image 5">
+                </div>
+            </div>
+
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <div
+                    style="
+                        width:100%;
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
```

Comment:
Is this image free from copyrights?

Reply:
This image is already getting used in previous example of carousel.
https://sap.github.io/fundamental-styles/?path=/docs/components-carousel--carousel-bottom.
Not sure on copy rights, but we have few more images in `assets/images/backgrounds/` folder.